servers/jishui/default.nix

```diff
@@ -0,0 +1,34 @@
+{ modulesPath, libPath, ... }:
+{
+  imports = [
+    "${modulesPath}/virtualisation/amazon-image.nix"
```

Comment:
You can use `serokell-nix.nixosModules.ec2` module now, it has apropriate configuration for ec2